website/versioned_docs/version-3.3.1/props/social_icons.md

```diff
@@ -199,9 +199,19 @@ title if made into a button (optional)
 
 social media type (required)
 
-|                                                                                                                                                            Type                                                                                                                                                            | Default |
-| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: |
-| social media type (angellist, codepen, envelope, etsy, facebook, flickr, foursquare, github-alt, github, gitlab, instagram, linkedin, medium, pinterest, quora, reddit-alien, soundcloud, stack-overflow, steam, stumbleupon, tumblr, twitch, twitter, google, google-plus-official, vimeo, vk, weibo, wordpress, youtube) |  none   |
+|                                                                                                                                                                           Type                                                                                                                                                                           | Default |
+| :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :-----: |
+| social media type (angellist, codepen, envelope, etsy, facebook, facebook-messenger, flickr, foursquare, github-alt, github, gitlab, instagram, linkedin, medium, pinterest, quora, reddit-alien, soundcloud, stack-overflow, steam, stumbleupon, tumblr, twitch, twitter, google, google-plus-official, vimeo, vk, weibo, whatsapp, wordpress, youtube) |  none   |
```

Comment:
The only thing I meant was that the 2 items should be added to this list. But not in the `versioned_docs` folder, but in the `docs` folder.